src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -195,8 +208,9 @@
                    (update-in [:wallet :transactions-unsigned] dissoc id)
                    (update-in [:wallet :send-transaction] merge clear-send-properties))}
           (if modal?
-            {:dispatch       [:navigate-back]
-             :dispatch-later [{:ms 400 :dispatch [:navigate-to-modal :wallet-transaction-sent-modal]}]}
+            (cond-> {:dispatch [:navigate-back]}
+                    (= method constants/web3-send-transaction)
+                    (assoc :dispatch-later [{:ms 400 :dispatch [:navigate-to-modal :wallet-transaction-sent-modal]}]))
```

Comment:
I know that it was not introduced here, but why this delayed dispatch ?

Reply:
idk maybe this is how modal window works, we need a delay between close and open modal windows

Reply:
i just remember it  worked wrong without delay